demo/item-userlists-model.ts

```diff
@@ -0,0 +1,33 @@
+export interface userlistDataInterface {
+  name: string;
+  item_is_member: boolean;
+  id: string;
+}
```

Comment:
I think the current schema uses `list_name` instead of `name`. (Though perhaps there's an argument for changing that)

Reply:
As old DB person, it was a rule not to have redundant naming with `[tablename.columnname]`, but I've seen the opposite rule :) i.e., every columnname is prefixed with the tablename. 

Rule of thumb: Prefer simplicity but avoid vagueness. Think `name` is just fine. But this particular sample data is OK if different from actual, however it ends up.